cmd/kola/options.go

```diff
@@ -50,3 +57,20 @@ func init() {
 	sv(&kola.AWSOptions.InstanceType, "aws-type", "t1.micro", "AWS instance type")
 	sv(&kola.AWSOptions.SecurityGroup, "aws-sg", "kola", "AWS security group name")
 }
+
+// Sync up the command line options if there is dependency
+func syncOptions() error {
+	for k, v := range kolaDefaultImages {
+		if kola.QEMUOptions.Board == k {
+			// check if DiskImage has been changed
+			if kola.QEMUOptions.DiskImage == kolaDefaultImages[defaultTargetBoard] {
```

Comment:
I'd rather make the default for the option blank so it isn't ambiguous whether the user specified the path or it really is the default value.